src/fsc/fsc.targets

```diff
@@ -52,6 +72,21 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" />
+
+    <!-- Reference MIBC optimization assemblies -->
+    <PackageReference Include="@(MIBCPackage)" />
```

Comment:
So, few questions
1. Do we upload one nuget package for all platforms now or one per platform?
2. If one, which profile will end up there?
3. If multiple, does sdk download all of them or one?

We need to make sure that if we upload one package, we pack all profiles there, and if we upload multiple, each has correct profile.

Reply:
Answered by Kevin in a different thread, resolving